Assets/MixedRealityToolkit-SDK/Features/UX/Scripts/Collections/CollectionEnums/OrientTypeEnum.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+namespace Microsoft.MixedReality.Toolkit.SDK.UX.Collections
+{
+    /// <summary>
+    /// Orientation type enum for collections
+    /// </summary>
+    public enum OrientTypeEnum
```

Comment:
ObjectCollectionOrientationType? Or, if more generic than that, OrientationType and move to a common location

Reply:
Sounds pretty generic.